website/docs/src/content/cli-reference/index.mdx

```diff
@@ -17,29 +17,65 @@ npm install -g gensx
 
 ## Available commands
 
+### Auth
 | Command                    | Description                      |
 | -------------------------- | -------------------------------- |
 | [`gensx login`](./login)   | Log in to GenSX Cloud            |
+
+### Development
+| Command                    | Description                      |
+| -------------------------- | -------------------------------- |
 | [`gensx new`](./new)       | Create a new GenSX project       |
 | [`gensx start`](./start)   | Start a local development server |
 | [`gensx build`](./build)   | Build a workflow for deployment  |
+
+### Deployment & Execution
+| Command                    | Description                      |
+| -------------------------- | -------------------------------- |
 | [`gensx deploy`](./deploy) | Deploy a workflow to GenSX Cloud |
 | [`gensx run`](./run)       | Run a workflow on GenSX Cloud    |
 
-## Basic usage
+### Environment Management
+| Command                          | Description                                  |
+| -------------------------------- | -------------------------------------------- |
+| [`gensx env create`](./env/create) | Create a new environment                    |
+| [`gensx env ls`](./env/ls)         | List all environments for a project          |
+| [`gensx env select`](./env/select) | Select an environment as active             |
+| [`gensx env unselect`](./env/unselect) | Unselect the current environment           |
+| [`gensx env show`](./env/show)     | Show the current environment details        |
 
+## Common Workflows
+
+### Starting a New Project
 ```bash
+# Log in to GenSX Cloud
+gensx login
+
 # Create a new project
 gensx new my-project
-
-# Navigate to project
 cd my-project
 
-# Start local development server
+# Start local development
 gensx start src/workflows.tsx
+```
+
+### Managing Environments
+```bash
+# Create and switch to a development environment
+gensx env create dev
```

Comment:
Should `create` automatically select the env? Feels liek the default we would want.

Reply:
The goal here was to strike the right middle ground between having the CLI be stateful and giving users explicit control. If we had this select the env, we'd be automatically opting people into the statefulness. 

One option would be to add a `--select` option as a middle ground.

Reply:
Both explicit select and --select flag feels really painful here. The 99% case when you create an environment is that you want to immediately use it. 

The only time someone could potentially ever get surprised is when you `deploy` or `run` and we prompt you to confirm anyways right?

Reply:
When we chatted about this in slack last week, one of the principles we agreed on in the thread was: "We never 'select' an environment on your behalf, you have to do it explicitly". I'll make the update if you want but it would go against what we had previously agreed on.

Reply:
That was not my takeaway. no surprises and no mutating anything production related (`gensx deploy` `gensx run`) based on implcit env config, just prompt the user to confirm.

Reply:
I copied that verbatim from the thread :)

Anyways, i will go make the update.

Reply:
Apologies for attempting to rewrite history 😆 

Thanks for making the change